server/src/main/resources/reference.conf

```diff
@@ -16,8 +17,8 @@
   }
   ssl {
     enabled=false
-    keyStorePassword=password
-    keyStorePath="/path/to/your/cert"
+    keyStorePassword=changeit
```

Comment:
**Minor:**
Usually we use _changeme_ 😉

Reply:
`changeit` I think is the default pwd for the java cert store

Reply:
yup, changeit is default usually for cert store

Reply:
Sorry, did not know 😊